src/devices/Directory.Build.props

```diff
@@ -5,7 +5,7 @@
   <Import Project="../../eng/Versions.external.props" />
   <PropertyGroup>
     <!-- Most projects should use this value to build for all possible target frameworks -->
-    <DefaultBindingTfms>net5.0;netcoreapp3.1;netstandard2.0;</DefaultBindingTfms>
+    <DefaultBindingTfms>net6.0;netcoreapp3.1;netstandard2.0;</DefaultBindingTfms>
```

Comment:
This will now cause that a .NET 5.0 app will fall back to the netstandard2.0 lib. Not sure we want that.

Reply:
Not really, they will fallback to the netcoreapp3.1 asset, and I checked with package validation and there is no loss on API  when they do that as all of the bindings that we have for net5.0 also work with netcoreapp3.1.